ttax/ops.py

```diff
@@ -108,3 +130,138 @@ def matmul(a, b):
   )
   func = to_function(tt_einsum)
   return func(a, b)
+
+
+def add(tt_a, tt_b):
+  """Returns a TensorTrain corresponding to elementwise sum tt_a + tt_b.
+  The shapes of tt_a and tt_b should coincide.
+  Supports broadcasting:
```

Comment:
Supports broadcasting, e.g. you can add a tensor train with batch size 7 and a tensor train with batch size 1:
  tt_batch.add(tt_single.batch_loc[np.newaxis])
where tt_single.batch_loc[np.newaxis] creates a singleton batch dimension.

Reply:
FYI Dim: np.newaxis is just None, but looks a bit more readable.